ios-base/DroidKaigi 2020/Extentions/NSLayoutConstraint+Ex.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+extension NSLayoutConstraint {
+    static func activate(_ constraints: NSLayoutConstraint...) {
```

Comment:
In my opinion.
Only convenience of this extension is that `[ ]` can be omitted.
I think it's better to call `NSLayoutConstraint.activate([...])`.
What do you think about this ? 🤔

Reply:
In my project, `multiline_literal_brackets` and `multiline_parameters_brackets` rules of SwiftLint are enabled.
Then, the code like
```swift
NSLayoutConstraint.activate([
    constraint1,
    constraint2
])
```
will trigger warning and expected to be
```swift
NSLayoutConstraint.activate(
    [
        constraint1,
        constraint2
    ]
)
```
so useless lines and indents are increased.

When omit `[ ] ` with this extension, it won't trigger warning.
That's why I made this extension.

Reply:
[multiline_literal_brackets](https://realm.github.io/SwiftLint/multiline_literal_brackets.html) and [multiline_parameters_brackets](https://realm.github.io/SwiftLint/multiline_parameters_brackets.html) are disabled by default on this project.
You can write like this.

```
NSLayoutConstraint.activate([
    iconImageView.heightAnchor.constraint(equalToConstant: 60),
    speakerView.topAnchor.constraint(equalTo: speakerViewContainer.topAnchor),
    speakerView.bottomAnchor.constraint(equalTo: speakerViewContainer.bottomAnchor),
    speakerView.leadingAnchor.constraint(equalTo: speakerViewContainer.leadingAnchor),
    speakerView.trailingAnchor.constraint(equalTo: speakerViewContainer.trailingAnchor),
    speakerView.topAnchor.constraint(equalTo: speakerButton.topAnchor),
    speakerView.bottomAnchor.constraint(equalTo: speakerButton.bottomAnchor),
    speakerView.leadingAnchor.constraint(equalTo: speakerButton.leadingAnchor),
    speakerView.trailingAnchor.constraint(equalTo: speakerButton.trailingAnchor)
])
```

Could you please change to use default method ?